docs/Quickstart-Guide-for-Beginners.md

```diff
@@ -0,0 +1,160 @@
+# Quickstart Guide for Beginners
+The aim of this page is enabling you to get the StartLeft application installed in your machine so that you can execute
+some commands, set up the REST API and, in summary, familiarize yourself with the usage of the tool.
+
+## Prerequisites
+
+---
+* Install the **[latest version of Python](https://www.python.org/downloads/)**.
+* Install **[pip3](https://pip.pypa.io/en/stable/installation/)**.
+* Install **[git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).**
+
+*During this guide some files will be downloaded or generated, so you can optionally create a folder to keep them
+organized.*
+
+## Install StartLeft
+
+---
+Install the last stable version of the tool using pip:
+```shell
+pip install git+https://github.com/iriusrisk/startleft.git
+```
+
+Check your installation with the command:
+```shell
+startleft --version
+```
+
+That should return something like:
+```shell
+startleft, version <your-installed-version>
+```
+
+## The parse command
+
+---
+The `parse` is the main command of the StartLeft CLI that enables you to perform the whole process of conversion
+to OTM by providing one or multiple source files and a mapping file. It presents slight variations depending on the type 
+of input source that you can check in the [CLI](usage/Command-Line-Interface.md) section or in each specific processor 
+documentation, but, to get an idea of its behavior, you can perform a conversion from an IaC file:
+
+First of all, let's download one of the examples contained in the StartLeft's `examples/terraform` folder:
+
+```shell
+wget https://raw.githubusercontent.com/iriusrisk/startleft/main/examples/terraform/multinetwork_security_groups_with_lb.tf
```

Comment:
Maybe we should create a dedicated folder to store the examples for this guide, so when they need to be changed for some reason we do not forget to check if the guide is still up to date, and they still work as explained.

Reply:
May be a good idea. Another problem of this approach is that we always are referring to the example in the main branch, while it should be from the same branch that the documentation.

Anyway, I do not want to stop this task because of this. I will include a note for further tasks to take this into account and acomplish the refactor.